### PR TITLE
David/qubit tracking

### PR DESCRIPTION
**Pull request by DavidSarlle**
This PR contains modifications in the iqm5q and qw5q runcards in order to accommodate the parameters needed for flux routines fitting in qibocal PR https://github.com/qiboteam/qibocal/pull/519

Qibolab PR https://github.com/qiboteam/qibolab/pull/581 should be merge before, in order to be able to access the new runcard paramters from the qubit object and be able to update them after fitting.

The PR also fixes some small bugs found in iqm5 platform and the config needed for the new LO controlling the TWPA (replaced ERA LO by a R&S LO).

New characterization for q0 in iqm5q (assg. fidelity q0: 93.3%).

imagen

imagen

This PR can be merged as it is, without waiting for merging the qibocal PR (https://github.com/qiboteam/qibocal/pull/519).
The qibolab PR.

### CODE REVIEW

File: iqm5q.py
Comment (Jacfomg) [inline iqm5q.py:15]:
```suggestion
TWPA_ADDRESS = "192.168.0.32" 
```

Let's remove the old IP

File: iqm5q.py
Comment (Jacfomg) [inline iqm5q.py:178]:
```suggestion

```
dead code

**Review by Jacfomg — CHANGES_REQUESTED** [review]
As you want to remove couplers this should be merged after #49.

**Comment by DavidSarlle** [comment]
> As you want to remove couplers this should be merged after #49.

I am removing the couplers because the platform in main and the runcard wasnot working with qibolab main. 

I think we should merge first this one and then add the info regarding the couplers. 

Are the parameters of your PR regarding the couplers and 2 qubit gates good? I suppouse that if we recharacterize the 5q, these information will be also modified...

File: iqm5q.py
Comment (DavidSarlle) [reply iqm5q.py:178]: done

File: iqm5q.py
Comment (DavidSarlle) [reply iqm5q.py:15]: done

**Comment by DavidSarlle** [comment]
added characterization of q2 with assigment fidelity of 86.1% (fine tunning still pending, but isbetter than what is now in main)

image

**Comment by Jacfomg** [comment]
> > As you want to remove couplers this should be merged after #49.
> 
> I am removing the couplers because the platform in main and the runcard wasnot working with qibolab main.
> 
> I think we should merge first this one and then add the info regarding the couplers.
> 
> Are the parameters of your PR regarding the couplers and 2 qubit gates good? I suppouse that if we recharacterize the 5q, these information will be also modified...

They will be conflicting with https://github.com/qiboteam/qibolab/pull/575 and https://github.com/qiboteam/qibolab_platforms_qrc/pull/49 on how coupler based devices should have the calibration runcards

**Comment by DavidSarlle** [comment]
added characterization of q2 with assignment fidelity of 75.1% (fine tunning still pending)

Untitled

**Comment by DavidSarlle** [comment]
The tests are not passing until we merge qibolab PR https://github.com/qiboteam/qibolab/pull/581 where the new qubit attributes has been introduced. As soon as we merge it, the test will pass.

**Comment by stavros11** [comment]
> The tests are not passing until we merge qibolab PR [qiboteam/qibolab#581](https://github.com/qiboteam/qibolab/pull/581) where the new qubit attributes has been introduced. As soon as we merge it, the test will pass.

For this we need to merge [qiboteam/qibolab#581](https://github.com/qiboteam/qibolab/pull/581) and **release** qibolab, because the workflow is using the stable release (to make sure that the default cluster environment works). In #56 I added a workflow that is using qibolab main to check such situations before releasing.

But we can still merge this when it's ready. I am hoping to make a qibolab release by the end of this week anyway.

**Comment by DavidSarlle** [comment]
> > The tests are not passing until we merge qibolab PR [qiboteam/qibolab#581](https://github.com/qiboteam/qibolab/pull/581) where the new qubit attributes has been introduced. As soon as we merge it, the test will pass.
> 
> For this we need to merge [qiboteam/qibolab#581](https://github.com/qiboteam/qibolab/pull/581) and **release** qibolab, because the workflow is using the stable release (to make sure that the default cluster environment works). In #56 I added a workflow that is using qibolab main to check such situations before releasing.
> 
> But we can still merge this when it's ready. I am hoping to make a qibolab release by the end of this week anyway.

Thanks @stavros11 for the comments. From my side is ready. The characterization for 1q gates is done for q0, q1 and q2. The q3 and q4 are not well characterized becasue the fridge is not cold. 
The chip is not operative so, up to you, we can merge or wait until the new release of qibolab

**Comment by stavros11** [comment]
@DavidSarlle @Jacfomg since most interface changing PRs in qibolab have been merged, we should probably also merge this and #49 to sync with the changes. Of course the calibration doesn't matter anymore, but it will be helpful to have these runcards in main so that we are compatible with qibolab-main and we can start the recalibration without bugs.

My plan is to merge this first as it is introducing the new parameters and then merge #49 to sync with the coupler changes. Do you agree?

**Comment by DavidSarlle** [comment]
I agree @stavros11 . Thanks.

**Review by stavros11 — APPROVED** [review]

**Review by andrea-pasquale — APPROVED** [review]

**Pull request merged**
